### ISSUE TITLE
Link missing a text alternative: website on collections index page

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5340]
Sites identified: MCO
Page(s) identified:

- Collections index page (display of website URL provided by collection manager)

WCAG compliance level: A

Description:

Links should always have a text alternative. The link text can be inserted into the HTML — it does not need to be visible to pass this check.

Links represented by images or icons can be problematic for visitors who are blind or have low vision.

The text alternative should describe the link's purpose — or what visitors can expect to find after they click it.

See more detail at a [SiteImprove example](https://my2.siteimprove.com/Inspector/1167098/A11Y/Page?pageId=37960016061&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r11/failed/K2NVS09vQlg=;Y2U3ZmQ3ZjA=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbXRnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b1db21510e69c76eac7249d51063b84687c858fd/image.png)

NOTE: In all the examples provided, there isn't actually a URL for the collection, but the "Website:" header is displaying with a blank space. Something about that is triggering the scanner to expect a URL.

We should also develop either a generic link text for these collection URLs (if we don't already have one), or ask collection managers to enter a link text when adding a URL (or remove the feature altogether!).

# Done Looks Like
- [ ] Website listing should not appear in the markup if no URL is set for a collection's website (resolving the empty value accessibility issue)

**Comment by joncameron** [comment]
Update the partial for a stronger check on website URL existence.

**Comment by elynema** [comment]
Some other examples of collections where this is happening:
https://media.dlib.indiana.edu/collections/nv935m97d
https://media.dlib.indiana.edu/collections/00000m29p

**Comment by elynema** [comment]
I tested this by deleting the URL for a collection that also had a website label. This caused the whole "Website:" element to disappear even though the label field value still existed. I also tried a website URL that was a blank space for the collection. The "Website:" element did no appear in that case. There is certainly on empty <a href /> showing up like there is for those examples in production.

I *think* this will resolve the issue in MCO, although we need to test there to see.
